packages/ERTP/src/typeGuards.js

```diff
@@ -0,0 +1,51 @@
+import { assert } from '@agoric/assert';
+import { Nat } from '@agoric/nat';
+
+/**
+ * Type guard for SetValue
+ *
+ * @param {Value} value
+ * @returns {value is SetValue}
+ */
+export const isSetValue = value => Array.isArray(value);
+
+/**
+ * Type guard for NatValue
+ *
+ * @param {Value} value
+ * @returns {value is NatValue}
+ */
+export const isNatValue = value => {
+  if (isSetValue(value)) {
+    return false;
+  }
+  return typeof Nat(value) === 'bigint';
+};
+
+/**
+ * @param {NatMathHelpers | SetMathHelpers } mathHelpers
+ * @returns {mathHelpers is SetMathHelpers}
+ */
+export const isSetMathHelpers = mathHelpers =>
+  isSetValue(mathHelpers.doMakeEmpty());
```

Comment:
This one assumes some very strong preconditions on its inputs. As a defensive type guard, this is trivial to fool
```js
const badSetHelpers = {
  doMakeEmpty: () => [], 
  doAdd: /*arbitrary malicious code including post-validation mutation of this non-frozen object*/,
};
isSetMathHelpers(badSetHelpers) // returns true
```
There are only two mathHelpers and we statically know them. 

```suggestion
export const isSetMathHelpers = mathHelpers => 
  mathHelpers === setMathHelpers;
```

Reply:
Oh, it turns out these aren't being used, so we can delete them.